### PR TITLE
feat: Upgrade plugin-sdk to v4

### PR DESCRIPTION
**Pull request by maaarcelino**
Hi there! This is a PR to update to our latest SDK! We are also rolling out hub shortly - https://hub.cloudquery.io/ so, once this PR is approved we can follow-up with additional PR so publish the plugin to our hub so users can enjoy dynamic searchable documentation for all versions and all tables.

### CODE REVIEW

**Comment by maaarcelino** [comment]
CC @Blaumaus

File: resources/live_visitors.go [hunk 1]
```diff
@@ -13,15 +13,5 @@ func LiveVisitors() *schema.Table {
 		Resolver:  fetchLiveVisitors,
 		Transform: transformers.TransformWithStruct(rest.LiveVisitors{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```
Comment (maaarcelino) [inline resources/live_visitors.go:18]: Removing as https://docs.swetrix.com/statistics-api#get-v1loglivevisitors doesn't return a `project_id` or any unique key in the response

File: resources/log.go [hunk 2]
```diff
@@ -13,15 +13,5 @@ func Log() *schema.Table {
 		Resolver:  fetchLog,
 		Transform: transformers.TransformWithStruct(rest.Log{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```
Comment (maaarcelino) [inline resources/log.go:18]: Removing as https://docs.swetrix.com/statistics-api#get-v1log doesn't return a `project_id` or any unique key in the response

File: resources/performance.go [hunk 3]
```diff
@@ -13,15 +13,5 @@ func Performance() *schema.Table {
 		Resolver:  fetchPerformance,
 		Transform: transformers.TransformWithStruct(rest.Performance{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```
Comment (maaarcelino) [inline resources/performance.go:18]: Removing as https://docs.swetrix.com/statistics-api#get-v1logperformance doesn't return a project_id or any unique key in the response

**Review by Blaumaus — APPROVED** [review]

**Comment by maaarcelino** [comment]
@Blaumaus can you approve the workflow and merge the PR if it's green, please?

I've created a team for you on our Hub. Can you give me an email address for you/someone on your side so I can transfer the ownership? You can share it here or email me at [marcel@cloudquery.io](mailto:marcel@cloudquery.io).

**Review by Blaumaus — APPROVED** [review]

**Comment by Blaumaus** [comment]
> @Blaumaus can you approve the workflow and merge the PR if it's green, please?
> 
> I've created a team for you on our Hub. Can you give me an email address for you/someone on your side so I can transfer the ownership? You can share it here or email me at [marcel@cloudquery.io](mailto:marcel@cloudquery.io).

Hi Marcel, thank you so much for maintaining this plugin😅

I've sent you an email from a swetrix.com domain with an email the plug-in should be transferred to.

Thank you.

**Comment by Blaumaus** [comment]
@maaarcelino also the PR workflow failed.. although I merged it, would it be possible to fix it in another PR?

**Comment by maaarcelino** [comment]
@Blaumaus No problem. I've submitted a PR to fix the issue: https://github.com/Swetrix/cq-source-swetrix/pull/5

**Pull request merged**